app/ts/components/subcomponents/coins.tsx

```diff
@@ -194,19 +194,19 @@ export function Token(param: TokenParams) {
 	</table>
 }
 
-export type TokenOrEtherParams = TokenParams | EtherParams | ERC721TokenParams
+export type TokenOrEtherParams = TokenParams | EtherParams | Erc721TokenParams
 
 export function TokenOrEth(param: TokenOrEtherParams) {
 	if ('decimals' in param) {
 		return <Token { ...param }/>
 	}
 	if ('id' in param) {
```

Comment:
ERC1155 also have ID, but for now you just adding 721 right?

Reply:
we already have 721, this PR does not add any asset types, just a naming refactor